CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## master / unreleased
 
+* [ENHANCEMENT] Return server side performance metrics for query-frontend (using Server-timing header). #3685
```

Comment:
CHANGELOG entries are grouped by type. Please move this below, along with the other `[ENHANCEMENT]`.